contracts/gateway/OSTPrime.sol

```diff
@@ -146,9 +146,9 @@ contract OSTPrime is UtilityToken, OSTPrimeConfig, Mutex {
     /* External functions. */
 
     /**
-     * @notice Convert the OST Prime token to OST Prime base token.
+     * @notice Convert the OST to coin.
```

Comment:
Maybe stay in line with the function name?
```suggestion
     * @notice Unwraps OST to coin.
```

Reply:
Is this intuitive to the reader that `unwraps` here means converting `ost` to `coin`?  🤔

Reply:
Good point. We could do it the "go" way and say "Unwrap converts OST to coin." Or you can leave it.